src/PrescriptionUtils.jl

```diff
@@ -19,8 +20,13 @@ export
   makeprescriptions,
   predictoutcomes
 
-ScikitLearn.@sk_import ensemble: RandomForestRegressor
-ScikitLearn.@sk_import linear_model: LinearRegression
+# @sk_import creates a global pointer that doesn't work with precompilation
+# copy directly: https://github.com/cstjean/ScikitLearn.jl/issues/50
+const ensemble = PyCall.PyNULL()
+
+function __init__()
+    copy!(ensemble, PyCall.pyimport("sklearn.ensemble"))
```

Comment:
With this change, I don't think we hit the code that auto installs scikit-learn

https://github.com/cstjean/ScikitLearn.jl/blob/9a8c424647099e33a895377a6ea4b089dc77cedd/src/Skcore.jl#L117

It seems like we should add `ScikitLearn.Skcore.import_sklearn()` as the first step in `__init__()`?

Reply:
We can change this line to `PyCall.pyimport_conda("sklearn.ensemble", "scikit-learn")` to auto install. It doesn't show the version issue as in the `import_sklearn` though, so probably should still add the line to call `import_sklearn()`

Reply:
That's why I suggested it :)